sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -22,85 +23,13 @@
 namespace Azure { namespace Security { namespace KeyVault { namespace _detail {
   class KeyVaultProtocolClient;
 }}}} // namespace Azure::Security::KeyVault::_detail
-
 namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
+  class KeyVaultRestoreDeletedSecretOperation;
+  class KeyVaultDeleteSecretOperation;
```

Comment:
Why?  These types should be public.  just include the header directly.

Look at the latest main for KeyVault keys, Operations is inside of the Models.hpp header and is just included in the client